javascript/sdk/src/index.ts

```diff
@@ -80,12 +82,12 @@ export class ForeverVM {
     return await this.#get('/v1/whoami')
   }
 
-  async createMachine(): Promise<CreateMachineResponse> {
-    return await this.#post('/v1/machine/new')
+  async createMachine(request?: CreateMachineRequest): Promise<CreateMachineResponse> {
+    return await this.#post('/v1/machine/new', request)
   }
 
-  async listMachines(): Promise<ListMachinesResponse> {
-    return await this.#get('/v1/machine/list')
+  async listMachines(request: ListMachinesRequest = {}): Promise<ListMachinesResponse> {
```

Comment:
Is there a reason we have a default value here but not for `createMachine`?

Reply:
it wasn't needed on `createMachine` because of we allow the body to be omitted in the createMachine request for backwards-compat, while the `listMachines` POST endpoint is new and so requires a body.

I like keeping things consistent though; changed `createMachine` to have a default.